### PR TITLE
Test

### PR DESCRIPTION
**Pull request by jericolopez1993**

### CODE REVIEW

**Comment by kennethcalamay** [comment]
Hi Jericho, nice to see a PR from you. I'll be looking at your code now.

File: app/concepts/todo/create.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 class Todo::Create < Trailblazer::Operation
-
+  contract do
+    property :title, validates: {presence: true}
+    property :description, validates: {presence: true}
+  end
```
Comment (kennethcalamay) [inline app/concepts/todo/create.rb:5]: Contract looks nice! 👍

File: spec/concepts/todo/create_spec.rb [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+RSpec.describe Todo::Create, type: :operation do
   it "can be spec'd" do
-    raise "Implement me"
+    puts "Implement me"
   end
+
+  it "Todo Created" do
+    todo = Todo::Create.(todo: {title: "Operation rules!", description: "Operation rules!"}).model
+    expect(todo.title).to eq("Operation rules!")
+    expect(todo.description).to eq("Operation rules!")
```
Comment (kennethcalamay) [inline spec/concepts/todo/create_spec.rb:11]: Impressive!

**Comment by jericolopez1993** [comment]
Hi, don't expect to much. I'm not confident on the result

**Comment by kennethcalamay** [comment]
It's a good start. You declared a contract but you aren't using it yet.
i.e. you're not calling `validate` from within your operation. Can you do that and push your changes afterwards?

**Comment by jericolopez1993** [comment]
Ok

**Comment by jericolopez1993** [comment]
Pushed

**Comment by kennethcalamay** [comment]
Nice. Looking good. Now your `process` method is too crowded. It's trying to do a lot of things. Can you determine what its 3 main responsibilities are and refactor them into different methods? Then push your changes.

**Comment by jericolopez1993** [comment]
Pushed

**Comment by jericolopez1993** [comment]
Pushed

**Comment by kennethcalamay** [comment]
You haven't extracted the three responsibilities of the `process` method into different methods yet. Can you do that and push your commits again?

**Pull request closed without merging**